src/Protocol/Models/BooleanOr.cs

```diff
@@ -15,7 +15,7 @@ public BooleanOr(bool value)
             _bool = value;
         }
 
-        public bool IsValue => this._value != default;
+        public bool IsValue => !this._value.Equals(default);
```

Comment:
@mholo65 @david-driscoll same with this...

Reply:
I reverted this... but I don't know why it's happening and don't think it's related to my change.